### ISSUE TITLE
No debug for NativeTlsClient

### ISSUE THREAD

**Issue by JuanPotato** [sfackler/hyper-native-tls#16]
Is there a specific reason why it doesn't have a debug trait implemented? I know the direct reason is probably `TlsConnector` doesn't have debug. But is it just because somewhere along the line of structs, one didn't have debug and thus stopped the others from deriving debug automatically and people just didn't care to implement debug? I'm just curious, not really an issue as I don't _need_ it, but it was just something I noticed.

**Comment by sfackler** [comment]
Nope, it's just an oversight I think. Feel free to make a PR, or I can add it in later today otherwise!
